guides/common/modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc

```diff
@@ -0,0 +1,27 @@
+[id="configuring-selinux-to-ensure-access-on-custom-ports_{context}"]
```

Comment:
file name and heading does not match the anchor.

Reply:
Thanks for review, fixed now?